src/Makefile.am

```diff
@@ -101,8 +101,8 @@ HUGE=tool_hugehelp.c
 if USE_MANUAL
 # Here are the stuff to create a built-in manual
 
-$(MANPAGE):
-	cd $(top_builddir)/docs && $(MAKE)
+# $(MANPAGE) exists and is up-to-date because docs is built before src
+# in SUBDIRS of root Makefile.am
```

Comment:
I want to be able to build from within the src/ dir so I think its actually better to set `../docs` as a build subdir in src/Makefile.am, and remove docs as a build subdir from the main Makefile.am.